### PR TITLE
Stacking strategy card UIs

### PR DESCRIPTION
**Pull request by Lonwyr**

### CODE REVIEW

**Comment by Lonwyr** [comment]
The UIs are now stacked with a slight z-offset and move up in case of a closed UI.

File: README.md [hunk 1]
```diff
@@ -41,3 +41,77 @@ Dependencies added with `yarn add -D {module}` (devDependenices) will **_not_**
...
+    .setWidth(/* width in px, optional, default 350 px */)
+    .setColor(new Color(1, 0, 0) /* ttpg-Color of the background */)
+    .register();
+```
+
+To send messages in the chat and color code the message in the players color a helper is available:
+
+```javascript
+const {
+    broadcastMessage,
```
Comment (darrellanderson) [inline README.md:84]: we should remove this and migrate the functionality to src/lib/broadcast

File: src/objects/strategy-cards/strategy-card-border.js [hunk 2]
```diff
@@ -81,21 +81,32 @@ class StrategyCardBorder extends Border {
...
+     * (Re)position the UI on the players desk. The <code>index</code> is used for
+     * the z-offset in case of multiple strategy card UIs on the same desk.
+     *
+     * @param {Integer} index Position of the UI on the stack on the players desk
+     */
+    positionUi(index) {
+        this._ui.position = this._desk.localPositionToWorld({
+            x: 30,
+            y: 0,
+            z: 20 + index * -1,
```
Comment (darrellanderson) [inline src/objects/strategy-cards/strategy-card-border.js:94]: You could also take a look at src/lib/ui/tabbed-panel, but this is good

**Review by darrellanderson — APPROVED** [review]

**Comment by darrellanderson** [comment]
I put some minor comments inline, I think moving to the src/lib/broadcast is a good idea so there aren't multiple versions of that but that can be a different change.

File: src/objects/strategy-cards/strategy-card-border.js (hunk 2 above)
Comment (Lonwyr) [reply src/objects/strategy-cards/strategy-card-border.js:94]: I considered that. In the end I wanted to focus the users attention more to work the strategy cards one by one in play order.

**Pull request merged**